src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -70,8 +75,14 @@ const Sidebar = () => {
           }`}
           onClick={selectPrivateChatTab}
         >
-          {hasUnreadMessages && <div className="notification"></div>}
+          {numberOfUnreadMessages > 0 && <div className="notification"></div>}
           <span>Messages</span>
+
+          {numberOfUnreadMessages > 0 && (
+            <span>
+              ({numberOfUnreadMessages}
+            </span>
+          )}
```

Comment:
There is another block of code above that shows a `div` with the `.notification` class. Should these be combined?

Reply:
Also, with the span here, was it just added so you can have a 'container' around the unread messages? Since we're not applying any styles/etc to it, we can probably just use a react fragment here:

- https://reactjs.org/docs/fragments.html